recipes/cmsis-pack-manager/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cmsis-pack-manager" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cmsis-pack-manager-{{ version }}.tar.gz
+  sha256: 35e506e8f148d9e4f0ab948db4007a64c035337cf526630d0f6f55f7f3b9b20c
+
+build:
+  entry_points:
+    - pack-manager = cmsis_pack_manager.pack_manager:main
+  script: {{ PYTHON }} -m pip install . -vv 
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - pip
```

Comment:
https://github.com/pyocd/cmsis-pack-manager/blob/66cca3956fee81b0be2dd88bd6c31aa07ecaa2b1/pyproject.toml#L2

These packages are required in the host requirements section!